src/backend/main/go.main/contact/contact.go

```diff
@@ -202,9 +202,5 @@ func FromVcard(user *objects.UserInfo, card vcard.Card) (*objects.Contact, error
 		}
 	*/
 
-	// Compute a title if none found
```

Comment:
why do you remove `helpers.ComputeNewTitle(contact)` ? I was suggesting to just remove the `if` around it

Reply:
This Title compute is done by CreateContact ;)